src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/OppfolgingstilfelleArbeidstakerSpek.kt

```diff
@@ -220,7 +220,7 @@ class OppfolgingstilfelleArbeidstakerSpek : Spek({
             result.shouldBeTrue()
         }
 
-        it("returns true if latest dialogmotekandidatEndringStoppunkt is after tilfelleEnd, latest DialogmoteKandidatEndring is not Kandidat, and tilfelleEnd is not before dialogmotekandidatStoppunktPlanlagt") {
+        it("returns false if latest dialogmotekandidatEndringStoppunkt is after tilfelleEnd, latest DialogmoteKandidatEndring is not Kandidat, and tilfelleEnd is not before dialogmotekandidatStoppunktPlanlagt") {
```

Comment:
For å få litt mindre kompleksitet, er det mulig/riktig å si "tilfelleEnd is after dialogmotekandidatStoppunktPlanlagt". Da slipper vi en negering. Nå har jeg ikke helt oversikt over om stoppunkt brukes andre steder, men tror det bare i denne forbindelsen. Trenger vi dialogmotekandidatendring foran stoppunkt?

Reply:
Skal forsøke å forbedre disse beskrivelsene 👍